src/size.ts

```diff
@@ -0,0 +1,45 @@
+import { each } from ".";
```

Comment:
In the current build process, there is a step to attach the `.js` extension, so it should be used as follows.
```suggestion
import each from "./each";
```

Reply:
thank you for your comment :) I changed it!